docs/reference/api.rst

```diff
@@ -4,22 +4,35 @@
 Flocker REST API Details
 ========================
 
+.. contents::
+	:local:
+
+Introduction
+============
+
 In general the API allows for modifying the desired configuration of the cluster.
 When you use the API to change the configuration, e.g. creating a new dataset:
 
 You will need to provide API end users with a certificate for authentication before they can use the API.
-Please see the :doc:`API authentication guide <./authentication>` for more information.
+For more information, see the :ref:`API authentication guide <authenticate>`.
 
 #. A successful response indicates a change in configuration, not a change to cluster state.
 #. Convergence agents will then take the necessary actions and eventually the cluster's state will match the requested configuration.
 #. The actual cluster state will then reflect the requested change.
-   E.g. cluster datasets state can be accessed via :http:get:`/v1/state/datasets`.
+   For example, getting the current cluster datasets can be accessed via :http:get:`/v1/state/datasets`.
```

Comment:
Still seems clumsy. "For example, the current cluster datasets can be retrieved via"? or "For example, you can retrieve teh curretn cluster datasets via".